### DISCUSSION TITLE
Anonymous form submissions are not compatible with sessions on this site

### DISCUSSION THREAD

**Discussion by HemboPagi** [pods-framework/pods#7090] [Q&A]
Hello all

I have spent some hours already to get rid of the the "Anonymous form submissions are not compatible with sessions on this site." message but no luck. 

I run a multisite with the latest PODS (2.9.15). POD is included via block editor.

I tried all the options under Sessions settings but non of the displayed the form. 

Any ideas?

**Comment by Clayton750** [comment]
I ended up using a registration plugin and redirecting the user to the registration form after a basic registration(on ultimate member) as there was no info on how to fix it.
